packages/react-native-codegen/src/parsers/__tests__/error-utils-test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @oncall react_native
```

Comment:
Sorry, I missed that before. Could you enable flow for the test file?
To do so, just update the license as it follows:
```suggestion
 * @flow strict-local
 * @oncall react_native
```

Reply:
Yes, just commited the suggestion